src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100, hidden = false) {
```

Comment:
you should only take `name` param here:
- all animals should have health of 100 BY DEFAULT
- only Herbivores can hide
```suggestion
  constructor(name) {
```